### ISSUE TITLE
Bar Chart Analysis of Model Runs for 5, "15"-Gene Networks

### ISSUE THREAD

**Issue by bklein7** [kdahlquist/GRNmap#274]
Bar charts will be used to assess modeling results generated in #265 per the guidelines specified during the Data Analysis Team meeting on Monday, November 7th:

1. Make two separate bar charts showing production_rate & threshold_b values generated for each transcription factor, clustered to show the results from all 5 networks (do not include dCIN5 results until @kdahlquist updates this network).

2. Make a bar chart showing the MSE values for each gene within the individual networks, clustered to show the different MSE values computed for each strain present within the network model.

**Comment by bklein7** [comment]
Moving forward, we will consider coding a function in MATLAB to produce these graphs as part of GRNmap. However, we will focus on producing these results in Excel for now to expedite this process.

**Comment by bklein7** [comment]
I created all requested bar charts and uploaded a PowerPoint presentation containing the results [here](https://github.com/kdahlquist/DahlquistLab/blob/master/documents/Five-Networks_Bar-Chart-Analysis_BK20161116.pptx).

**Comment by kdahlquist** [comment]
* generate new charts dealing with the threshold b issue #289 
* fix assorted labeling issues.

**Comment by bklein7** [comment]
The bar chart analysis [PowerPoint Presentation](https://github.com/kdahlquist/DahlquistLab/blob/master/documents/Five-Networks_Bar-Chart-Analysis_BK20161116.pptx) was updated to reflect the changes requested above. Specifically, threshold_b values for genes with no inputs (ABF1, GCR2, ZAP1) were adjusted to zero, and the language used in the chart legends was corrected.

I attempted to add the results from @Nwilli31's CIN5 model runs but could not find these output sheets posted in the Dahlquist Lab repository. Are they uploaded? And if so, should I include the data from both the 14 and 17 gene CIN5 networks?

**Comment by kdahlquist** [comment]
@Nwilli31 will need to respond about the location of the output sheets.  When they are posted, please do the analysis for both networks.

**Comment by Nwilli31** [comment]
I've uploaded the documents to the Dahlquist Repository. I can do the analysis for the two networks; however, they will not be on the same graph because I do not have an excel file of all the GRNmap output comparisons that @bklein7 may have created. Is this on the repository? If not, I can create a new document with all the information.

**Comment by bklein7** [comment]
I looked at @Nwilli31's updates to the bar chart analysis powerpoint and noticed some issues. Primarily, the corrections made to account for the erroneous threshold_b values (#289) were not carried forward in the new powerpoint. I presume this is because @Nwilli31 did not see my excel file with the bar chart data and made a new one (for reference, this file is in the [15-gene_networks_analysis](https://github.com/kdahlquist/DahlquistLab/tree/master/data/15-gene_networks_analysis) folder in the Dahlquist Lab repository). I have corrected for this issue and made a few other changes to the powerpoint, which is available [here](https://github.com/kdahlquist/DahlquistLab/tree/master/documents).
